README.mingw.md

```diff
@@ -11,21 +11,12 @@ steps are recommended:
    On Fedora, with `dnf install mingw32-gcc-c++`, or `dnf install mingw64-gcc-c++` for the
    64-bit Windows.
 
-3. Install cross-compiled dependency packages.  Alternatively see [^1] below.
-   On Fedora that would be `dnf install mingw32-glib2 mingw32-cairo mingw32-freetype`
-   for 32-bit, or `dnf install mingw64-glib2 mingw64-cairo mingw64-freetype` for 64-bit.
+3. `.ci/build-win32.sh`
```

Comment:
If we are going to recommend this to people, then `.ci` is not the most appropriate place for it.

Reply:
Maybe renaming `.ci` to something like `.maintaining` or something? I believe we shouldn't offer any of these as a regular part of the library and README.mingw.md is mostly useful for maintainers and not clients anyway, maybe removing README.mingw.md from repo and putting it somewhere less public like GitHub wiki pages.

Reply:
A correct point given it is used as a part of release process also but like to avoid putting them again somewhere that accessible.

Reply:
Up to you then.